src/components/visualization-bar/spec.js

```diff
@@ -0,0 +1,19 @@
+import { div } from 'utils/selection';
+
+import { VisualizationBar } from 'components/visualization-bar';
+
+export default () => {
+  describe('hx-visualization-bar', () => {
+    it('should create and class the progress bar correctly', () => {
+      const pb = new VisualizationBar(div());
+      pb.selection.classed('hx-visualization-bar').should.equal(true);
+      pb.selection.selectAll('.hx-progress-bar-inner').size().should.equal(1);
+      pb.progress.should.equal(0);
+      /*
+        This method is used rather than calculate the pixel width as .style('width')
+        returns Npx not N%
+      */
+      pb.innerBars.attr('style').split(' ').join('').should.equal('width:0%;');
```

Comment:
I'm moving gradually towards using [this brilliant JS testing guide](https://github.com/mawrkus/js-unit-testing-guide) so these tests will need updating

For instance, each expectation should be moved into its own test case:
```js
describe('when creating a bar with no options', () => {
  let thisSel;
  let thisVisualizationBar;

  beforeEach(() => {
    thisSel = div();
    thisVisualizationBar = new VisualizationBar(thisSel);
  });

  it('sets the class correctly', () => {
    thisSel.classed('hx-visualization-bar').should.equal(true);
  });

  it('creates a single bar', () => {
    thisSel.selectAll('.hx-progress-bar-inner').size().should.equal(1);
  });

  it('sets the progress to zero', () => {
    thisVisualizationBar.progress().should.equal(0);
  });

  it('sets the width of all bars to zero percent', () => {
    /*
    This method is used rather than calculate the pixel width as .style('width')
    returns Npx not N%
  */
  thisVisualizationBar._.innerBars.attr('style').split(' ').join('').should.equal('width:0%;');
  });
});
```

As a personal preference, I omit `should` from the start of each test case. There's no point in adding it everywhere as it just pollutes the test output. (i.e. `when creating a bar with no options > sets the class correctly` reads just as well as `when creating a bar with no options > should set the class correctly`)